pkg/controller/kibana/pod.go

```diff
@@ -173,12 +174,54 @@ func NewPodTemplateSpec(
 			WithInitContainers(keystore.InitContainer)
 	}
 
+	if kb.Spec.PackageRegistryRef.IsDefined() {
+		builder, err = withEPRCertsVolume(builder, kb)
+		if err != nil {
+			return corev1.PodTemplateSpec{}, err
+		}
+	}
+
```

Comment:
```suggestion
	var additionalInitEnvVars []corev1.EnvVar
	if kb.Spec.PackageRegistryRef.IsDefined() {
		userNodeExtraCACerts := getUserNodeExtraCACerts(kb)
		builder, err = withEPRCertsVolume(
			builder,
			kb,
			userNodeExtraCACerts != "",
		)
		if err != nil {
			return corev1.PodTemplateSpec{}, err
		}
		if userNodeExtraCACerts != "" {
			// Pass original user path to init container so it can copy from there
			additionalInitEnvVars = append(additionalInitEnvVars, corev1.EnvVar{
				Name:  nodeExtraCACerts,
				Value: userNodeExtraCACerts,
			})
		}
	}
```